app/src/components/buttons/ghostButton/ghostButton.scss

```diff
@@ -90,6 +90,20 @@
       border-color: transparent;
       border-radius: 2px;
     }
+
+    &.transparent-border-hover {
```

Comment:
Why not use the same color as [here](https://github.com/reportportal/service-ui/pull/2466/files#diff-d77f56b743ba16c57f49e68360668a74R76-R87)?